plugins-builtin/src/main/scala/za/co/absa/enceladus/plugins/builtin/errorsender/mq/KafkaErrorSenderPluginImpl.scala

```diff
@@ -92,13 +92,19 @@ case class KafkaErrorSenderPluginImpl(connectionParams: KafkaConnectionParams,
 
     val allowedErrorCodes = KafkaErrorSenderPluginImpl.errorCodesForSource(params.sourceId)
 
+    val reportDateCol: Column = if (dataFrame.columns.contains(ColumnNames.reportDate)) {
```

Comment:
So is there an option for no ReportDate? When does that happen? Does the information still have the same informational value?

Reply:
Well, there are datasets where `ReportDate` is not present - e.g. Cobol ones, but in general, this is possible - the presence of this field is not forced in Enceladus in any way. Here, I only fixed the obvious error in that case.

Unless someone deliberately disables/mangles the `enceladus_record_id` generation, that alone is sufficient to uniquely identify the error origin. `ReportDate` is just a helpful field.

Reply:
The report Date that we add is, I think, Data Quality requirement. But this is not a place for that discussion. 👍